.github/workflows/update-deps.yml

```diff
@@ -3,11 +3,11 @@ name: Update Dependencies
 on:
   # Run every day.
   schedule:
-    - cron: '0 3 * * *'
+    - cron: "0 3 * * *"
   # And on on every PR merge so we get the updated dependencies ASAP, and to make sure the changelog doesn't conflict.
   push:
-    branches:
-      - main
+    # branches:
+    #   - main
```

Comment:
Are the deps updates disabled for main on purpose? I don't see why would we want that?

Reply:
no, just realized this morning I also need to install Flutter SDK during the updater workflow so I have it enabled on all pushes now while working on it.

sorry for the confusion